doc/api/async_hooks.md

```diff
@@ -2,7 +2,10 @@
 
 <!--introduced_in=v8.1.0-->
 
-> Stability: 1 - Experimental
+> Stability: 1 - Experimental. We plan to remove this API
+> in the future; it will never become stable.
+> Please migrate to [`AsyncLocalStorage`][] and [Diagnostics Channel][] and
+> other stable APIs.
```

Comment:
This is quite vague. I think we should either remove this or point to concreate APIs (which I'm not aware of).

Reply:
I am pointing to specific APIs. I don’t think we need any more detail here. This is a quick note informing people that this API will go away. It’s not our responsibility to tell them how to rewrite their code.

Reply:
@Flarna Would this be better?

```suggestion
> in the future; it will never become stable. If possible,
> please use other APIs instead such as [`AsyncLocalStorage`][] or
> [Diagnostics Channel][].
```